### PR TITLE
Cat people have feline screams now and custom emote

### PR DESCRIPTION
**Pull request by JamieD1**
### Intent of your Pull Request

Lizards have hiss so cats now have meow, makes sense :thonk:

# Changelog

:cl:  
rscadd: Felindids now have custom emotes
/:cl:

### CODE REVIEW

File: .vscode/settings.json [hunk 1]
```diff
@@ -7,12 +7,9 @@
 		"tgui/.yarn": true,
 		"tgui/.pnp.*": true
 	},
-	"workbench.editorAssociations": [
-		{
-			"filenamePattern": "*.dmi",
-			"viewType": "imagePreview.previewEditor"
-		}
-	],
+	"workbench.editorAssociations": {
```
Comment (Bibby0110) [inline .vscode/settings.json]: undocumented change

File: code/modules/mob/living/carbon/human/emote.dm [hunk 2]
```diff
@@ -75,18 +75,39 @@
...
+	if(iscatperson(H))
+		return pick('sound/voice/feline/scream1.ogg', 'sound/voice/feline/scream2.ogg')
+	else if(ishumanbasic(H))
 		if(user.gender == FEMALE)
 			return pick('sound/voice/human/femalescream_1.ogg', 'sound/voice/human/femalescream_2.ogg', 'sound/voice/human/femalescream_3.ogg', 'sound/voice/human/femalescream_4.ogg', 'sound/voice/human/femalescream_5.ogg')
 		else
 			if(prob(1))
 				return 'sound/voice/human/wilhelm_scream.ogg'
 			return pick('sound/voice/human/malescream_1.ogg', 'sound/voice/human/malescream_2.ogg', 'sound/voice/human/malescream_3.ogg', 'sound/voice/human/malescream_4.ogg', 'sound/voice/human/malescream_5.ogg')
+	else if(iscatperson(H))
```
Comment (Bibby0110) [inline code/modules/mob/living/carbon/human/emote.dm]: in here twice?

File: code/modules/mob/living/carbon/human/emote.dm [hunk 3]
```diff
@@ -75,18 +75,39 @@
...
+		return pick('sound/voice/feline/scream1.ogg', 'sound/voice/feline/scream2.ogg')
 	else if(ismoth(H))
 		return 'sound/voice/moth/scream_moth.ogg'
 	else if(H.dna?.species?.screamsound) //yogs start: grabs scream from screamsound located in the appropriate species file.
 		return H.dna.species.screamsound //yogs end - current added screams: lizard, preternis.
 
+/datum/emote/living/carbon/meow
```
Comment (Bibby0110) [inline code/modules/mob/living/carbon/human/emote.dm:92]: why not /datum/emote/living/carbon/human/meow

Comment (JamieD1) [reply code/modules/mob/living/carbon/human/emote.dm:92]: Copying hiss, can change that as well I guess

**Comment by Yogbot-13** [comment]
#### Test Merge Deployed

##### Server Instance
main

##### Merged By
JamieD12

##### Revision
Origin: 6eac6b8678728327d8878fa07e5c0f4655a8fa3e
Pull Request: 7e6ff9a6532068e58ace060d95ecf12116821014
Server: 4cf045dcc80de3751ed501a3fdf71a161b4b6078

**Comment by Yogbot-13** [comment]
#### Test Merge Updated

##### Server Instance
main

##### Merged By
JamieD12

##### Revision
Origin: 09dc7735c329443b1d7140a2d32a9b7e5388298f
Pull Request: 7e6ff9a6532068e58ace060d95ecf12116821014
Server: 8ce5531e4aa90f730f2881ca15125742069cc3b7

File: .vscode/settings.json [hunk 4]
```diff
@@ -7,12 +7,9 @@
 		"tgui/.yarn": true,
 		"tgui/.pnp.*": true
 	},
-	"workbench.editorAssociations": [
-		{
-			"filenamePattern": "*.dmi",
-			"viewType": "imagePreview.previewEditor"
-		}
-	],
```
Comment (Bibby0110) [inline .vscode/settings.json]: still undocumented change

**Comment by Yogbot-13** [comment]
#### Test Merge Updated

##### Server Instance
main

##### Merged By
JamieD12

##### Revision
Origin: 50e22d907e88fc4aed08712ae4939d0ca006b9bd
Pull Request: 7e6ff9a6532068e58ace060d95ecf12116821014
Server: cfbef971e39f756f57c6351ccba16176fb7ca9b7

**Comment by Yogbot-13** [comment]
#### Test Merge Updated

##### Server Instance
main

##### Merged By
JamieD12

##### Revision
Origin: d244ed8af6b4625f69aff980ca6f669c8eab96cc
Pull Request: 7e6ff9a6532068e58ace060d95ecf12116821014
Server: c9dd238bda1b13104771876231c7f061360f117a

**Comment by Yogbot-13** [comment]
#### Test Merge Updated

##### Server Instance
main

##### Merged By
JamieD12

##### Revision
Origin: b34fc836328b6ea784801f980d65536c09daad58
Pull Request: 7e6ff9a6532068e58ace060d95ecf12116821014
Server: 99a8136a2c3ea2386b7417cadce313eef7205037

**Comment by Yogbot-13** [comment]
#### Test Merge Updated

##### Server Instance
main

##### Merged By
JamieD12

##### Revision
Origin: 51d15f34c48d85520d7724f0b4ab5cd650e8be05
Pull Request: 7e6ff9a6532068e58ace060d95ecf12116821014
Server: 08a5b01226a862a638855658a1fcb349f2180bdf

**Comment by Yogbot-13** [comment]
#### Test Merge Updated

##### Server Instance
main

##### Merged By
JamieD12

##### Revision
Origin: faf404bfec32c473db5ed3f6fa8061fcaa8b01d8
Pull Request: 7e6ff9a6532068e58ace060d95ecf12116821014
Server: 261ac2b415670d9615de140a821f0aed25f45412

**Comment by Yogbot-13** [comment]
#### Test Merge Updated

##### Server Instance
main

##### Merged By
JamieD12

##### Revision
Origin: dfdf9d25fba679e22d2ccaca56dc3ff10960e5af
Pull Request: 7e6ff9a6532068e58ace060d95ecf12116821014
Server: a57b89a5cefc62a3059460567fc50d2b23efa5f5

**Comment by Yogbot-13** [comment]
#### Test Merge Updated

##### Server Instance
main

##### Merged By
JamieD12

##### Revision
Origin: dfdf9d25fba679e22d2ccaca56dc3ff10960e5af
Pull Request: 7e6ff9a6532068e58ace060d95ecf12116821014
Server: d723368c137b9977e75b3e26dfb321c788297cdd

**Comment by Yogbot-13** [comment]
#### Test Merge Updated

##### Server Instance
main

##### Merged By
JamieD12

##### Revision
Origin: 24d1f239ea0aa6c88caabde6243b7528fe97fb4c
Pull Request: 7e6ff9a6532068e58ace060d95ecf12116821014
Server: 1c84e2f8d3d922d24bcc92d7edb9ce9e32d5252e

**Comment by Yogbot-13** [comment]
#### Test Merge Updated

##### Server Instance
main

##### Merged By
JamieD12

##### Revision
Origin: e581b726a44943425eb38bb248e800e792d0fd9d
Pull Request: 7e6ff9a6532068e58ace060d95ecf12116821014
Server: fc7122cf9f35688fd96d1b459cc1c9b61682e752

**Comment by Yogbot-13** [comment]
#### Test Merge Updated

##### Server Instance
main

##### Merged By
JamieD12

##### Revision
Origin: 1befeb33968cd7fcd11341f081e90e149979098b
Pull Request: 7e6ff9a6532068e58ace060d95ecf12116821014
Server: 87f7ce98fbe6e2119e87704ba716dc7213e89bd9

**Comment by Yogbot-13** [comment]
#### Test Merge Updated

##### Server Instance
main

##### Merged By
JamieD12

##### Revision
Origin: bfcf8bfb1e922f1d6c8f6db19948a76bbfa39116
Pull Request: 7e6ff9a6532068e58ace060d95ecf12116821014
Server: e56275b82d8a1b0a3a432a233629ac206864f83a

**Comment by Yogbot-13** [comment]
#### Test Merge Updated

##### Server Instance
main

##### Merged By
JamieD12

##### Revision
Origin: 074dc717e2c59a75f46c232f5d068130b857d36d
Pull Request: 7e6ff9a6532068e58ace060d95ecf12116821014
Server: e10404c9a10c2ee7f917b59885fa18ede7f0435a

**Comment by Yogbot-13** [comment]
#### Test Merge Updated

##### Server Instance
main

##### Merged By
JamieD12

##### Revision
Origin: aeb34a71e3fd3563bcfadbc47b1b7574c1f9359b
Pull Request: 7e6ff9a6532068e58ace060d95ecf12116821014
Server: 3fc96da3fe7b326106f45bec9fcd29d2afa4b3c4

**Comment by Yogbot-13** [comment]
#### Test Merge Updated

##### Server Instance
main

##### Merged By
JamieD12

##### Revision
Origin: 5b96721222519db28316c53f605c7f8603965bfc
Pull Request: 7e6ff9a6532068e58ace060d95ecf12116821014
Server: 0e5c469765b215fb6c9c7efe7e82d2ab360f0212

**Review by adamsong — APPROVED** [review]
Code is fine, works and intended, good to merge once art reviews it

**Comment by Yogbot-13** [comment]
#### Test Merge Updated

##### Server Instance
main

##### Merged By
JamieD12

##### Revision
Origin: a2b0d80bb2fe24c63b5611d324b01f653679b0fa
Pull Request: 7e6ff9a6532068e58ace060d95ecf12116821014
Server: 48ff5736e8e42a68fceb7e361203ebf0fa1bbeb2

**Comment by Yogbot-13** [comment]
#### Test Merge Updated

##### Server Instance
main

##### Merged By
JamieD12

##### Revision
Origin: 75e241882ae3eba86957aa8ef47647e805b2a28b
Pull Request: 7e6ff9a6532068e58ace060d95ecf12116821014
Server: a573d4fda4232a30130441438b8607ed178df93e

**Comment by Yogbot-13** [comment]
#### Test Merge Updated

##### Server Instance
main

##### Merged By
JamieD12

##### Revision
Origin: b0848cac60e6718660d3154778f47593877aaf0e
Pull Request: 7e6ff9a6532068e58ace060d95ecf12116821014
Server: 46c827dc9eeae1ddda6382961b987de97833563a

**Review by nesisknesis — APPROVED** [review]

**Pull request merged**